channel-router/src/app.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import Koa from 'koa';
+import bodyParser from 'koa-bodyparser';
+import router from './routes';
+import { configuration } from './config';
+import { initializeRedis } from './clients/redisClient';
```

Comment:
These lines aren't being used. Which also makes me think we don't need all the other redis files in this repo